tests/testrom.c

```diff
@@ -299,9 +309,12 @@ int main() {
 		if (ctx.result == TEST_FAILED) {
 			failures++;
 			printf("FAIL\n\n");
-
 			if (ctx.log != logbuf) {
-				printf("%s\n\n", logbuf);
+				debugf("%s\n", logbuf);
+			}
+			if (ctx.err != errbuf) {
+				printf("%s\n", errbuf);
+				debugf("%s\n", errbuf);
```

Comment:
Was there a specific reason why we do `console_set_debug(false);` at start of tests?

Reply:
I think my general idea was that the test output on the screen wouldn't always match the console. On the screen we do something graphical formatting / alignment that is not required in the logs. Also maybe in the future we should probably switch to a slightly more advanced UI (eg: having a fixed header and a scrollable area, etc). So I started from the day one with this idea and kept the two separated. If you look at the code, there are already some differences, albeit not fundamental for now, I agree with that.